iosHackthonApp/iosHackthonApp/Resources/Colors/ColorManager.swift

```diff
@@ -10,11 +10,10 @@ import SwiftUI
 
 struct ColorManager {
 
-	static let appPrimary = Color("appPrimary")
-	static let appSecondary = Color("appSecondary")
-	static let red = Color.red
-	static let blue = Color.blue
-	static let gray = Color.gray
+    static let red = Color.red
+    static let blue = Color.blue
+    static let gray = Color.gray
     static let appGreen = Color("appGreen")
+    static let appDarkGreen = Color("appDarkGreen")
```

Comment:
Could the green and dark green be the primary and secondary?

Reply:
Sure.
That would also make more sense if we were to add Dark mode colours too - I'll change it